### PR TITLE
add CDN publish jobs

### PR DESCRIPTION
**Pull request by davidleger95**
Let me know if you see any tweaks that need to be made

### CODE REVIEW

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -25,3 +25,74 @@ jobs:
...
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}/LICENSE
+      - run:
+          gsutil -m cp ${GITHUB_WORKSPACE}/README.md
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}/README.md
+      - run:
+          gsutil -m cp ${GITHUB_WORKSPACE}/package.json
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}/package.json
+      - run:
+          # copy types only
+          gsutil -m cp -r ${GITHUB_WORKSPACE}/dist/types/v*
```
Comment (davidleger95) [inline .github/workflows/prerelease.yml]: hoping this works as expected (it does for the npm publish)

**Comment by domenicrosati** [comment]
> Let me know if you see any tweaks that need to be made

@davidleger95  what is a CDM

**Comment by domenicrosati** [comment]
:P

File: .github/workflows/prerelease.yml [hunk 2]
```diff
@@ -25,3 +25,74 @@ jobs:
...
+          gsutil -m cp ${GITHUB_WORKSPACE}/package.json
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}/package.json
+      - run:
+          gsutil -m cp -r ${GITHUB_WORKSPACE}/dist
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}
+      - run:
+          gsutil -m cp -r ${GITHUB_WORKSPACE}/loader
+          gs://${BUCKET}/${PACKAGE}@${GITHUB_REF#refs/tags/v}
+      # Note: DON’T publish a prerelease to the main folder like we would with a release!
+  cdn-types:
```
Comment (davidleger95) [inline .github/workflows/prerelease.yml]: A bit redundant to have a separate job for the types package but I'm a bit worried combining them would be too messy. Any thoughts?

**Comment by domenicrosati** [comment]
CDM: Continous Dance Musix

File: .github/workflows/prerelease.yml (hunk 1 above)
Comment (domenicrosati) [reply .github/workflows/prerelease.yml]: you can always give it a try

Comment (domenicrosati) [reply .github/workflows/prerelease.yml]: I too am uncertain about the * thing

File: .github/workflows/prerelease.yml (hunk 2 above)
Comment (dangodev) [reply .github/workflows/prerelease.yml]: I totally feel you, but I don’t know if there’s a way to re-use actions between workflows like you can in CircleCI

If something really needs to be re-used, then maybe it makes sense to have it as a Node Script. But as-is no I think this is fine for now. They’re not too hard to edit/maintain IMO

File: .github/workflows/prerelease.yml (hunk 1 above)
Comment (dangodev) [reply .github/workflows/prerelease.yml]: Does that work locally? e.g. `cp -r dist/types/v* fakefolder`? If so then you’re probably good.

**Comment by dangodev** [comment]
@davidleger95 try tagging this branch with `v0.0.1-test.0` or something and see what happens! 🤞

File: .github/workflows/prerelease.yml (hunk 2 above)
Comment (sslotsky) [reply .github/workflows/prerelease.yml]: I don't actually think we need our types on the CDN. Only the WC libraries are going to use them.

File: .github/workflows/prerelease.yml (hunk 1 above)
Comment (davidleger95) [reply .github/workflows/prerelease.yml]: yes, that does work locally. As sam said above though, we don't have to publish types to the cdn so I'll remove this

File: .github/workflows/prerelease.yml (hunk 2 above)
Comment (davidleger95) [reply .github/workflows/prerelease.yml]: Cool, I'll remove this then

**Pull request merged**